sound/soc/sof/ipc4-topology.c

```diff
@@ -709,11 +736,24 @@ static int sof_ipc4_widget_setup_comp_mixer(struct snd_sof_widget *swidget)
 
 	return 0;
 err:
+	sof_ipc4_free_audio_fmt(&mixer->available_fmt);
```

Comment:
optimization: you could replace those three lines by a call to sof_ipc4_widget_free_comp_mixer()

Reply:
> optimization: you could replace those three lines by a call to sof_ipc4_widget_free_comp_mixer()

@plbossart Got it. Thanks.